source/content/partials/traffic-overages-faq.md

```diff
@@ -10,7 +10,7 @@ reviewed: ""
 
 ### Will Pantheon warn me if my site has an overage?
 
-Customers are responsible for monitoring their usage and overage risk using the Site Dashboard. Pantheon periodically reviews customer usage for overages across all plan levels and will contact you directly by email. If the change to traffic behavior exceeds your plan limit for any two months of traffic, your site will be moved to the next appropriate plan to help avoid further overages. These adjustments will be communicated in an email with a subject of "You've been right-sized".
+Customers are responsible for monitoring their usage and overage risk using the Site Dashboard. Pantheon periodically reviews customer usage for overages across all plan levels and will contact you directly by email. If the change to traffic behavior exceeds your plan limit, your site may be subject to overages enforcement. Overages protection exists for Performance and Elite sites on an annual agreement. Refer to the [FAQ](https://docs.google.com/document/d/1vaLJHXJJashwO9zsHFeywW-RQx_6uWOWH2Yr31sbQv4/edit?usp=sharing) for more details on Overage Protection and what to expect should your site go over the plan limits.
```

Comment:
> your site may be subject to overages enforcement

That sounds unnecessarily vague. I think it is more accurate and specific to say   something like "you will be billed for additional visits unless your site is covered by overages protection."

Reply:
@carly-armstrong is that accurate

Reply:
@stevector @carly-armstrong We can say: 
 apply site visitor overages
 overage will be billed 
 pay overage fees